### PR TITLE
Update to v 0.10.0:

### PR DESCRIPTION
**Pull request by akabanovs**
- version and sha256 updated.
 - source file url updated.
 - patch added - needed for numpy >=1.20, pulled in implicitly.
 - build script updated.
 - build number reset to 0
 - dependencies updated.
 - pip check added to tests.

### CODE REVIEW

**Comment by sumit0190** [comment]
You can remove `packaging` from the run section, as it's a test dependency only. And you'd have to add `patch` (and `m2-patch` for Windows) in the build section for the recipe to be more reproducible.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% set name = "rbc" %}
-{% set version = "0.5.1" %}
+{% set version = "0.10.0" %}
 {% set pypi_name = "rbc-project" %}
 {% set bundle = "tar.gz" %}
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml]: this variable is not really needed as used only once

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {% set name = "rbc" %}
-{% set version = "0.5.1" %}
+{% set version = "0.10.0" %}
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml]: there is already 0.11.0 available

File: recipe/meta.yaml [hunk 3]
```diff
@@ -8,37 +8,41 @@ package:
...
     - python
     - thriftpy2
     - tblib
-    - llvmlite >=0.29
-    - numba
+    - llvmlite
+    - numba >=0.55
     - netifaces
-    - six
+    - packaging
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml]: it is in test_requirements but also in requirements.txt https://github.com/xnd-project/rbc/blob/f816e237ae8929ef472a83b668556f94600277f9/requirements.txt#L6 @sumit0190  ?

Comment (sumit0190) [reply recipe/meta.yaml]: It's in `requirements.txt`, but is only used in the `tests` directory; maybe that's a mistake on their end:
https://github.com/search?q=repo%3Axnd-project%2Frbc+packaging&type=code

**Comment by danpetry** [comment]
@akabanovs do you need help moving this forward?

**Comment by akabanovs** [comment]
Updated to 0.11.0. No patch is required for this version.

**Comment by akabanovs** [comment]
@danpetry thanks, I have already addressed the comments.

File: recipe/meta.yaml (hunk 2 above)
Comment (akabanovs) [reply recipe/meta.yaml]: Thanks, updated.

File: recipe/meta.yaml (hunk 1 above)
Comment (akabanovs) [reply recipe/meta.yaml]: Thanks

**Comment by akabanovs** [comment]
> You can remove `packaging` from the run section, as it's a test dependency only. And you'd have to add `patch` (and `m2-patch` for Windows) in the build section for the recipe to be more reproducible.

Yes, thanks, forgot to add that. However, v 0.11.0 does not require patch anymore

**Review by skupr-anaconda — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,44 +1,44 @@
...
-    - numba
+    - llvmlite
+    - numba >=0.55
     - netifaces
-    - six
 
 test:
   requires:
-    - pytest
+    # - pytest
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml]: remove that line, otherwise LGTM

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Pull request merged**